content/blog/posts/student-story-whitehet.rst

```diff
@@ -0,0 +1,54 @@
+Reflections on My Time at the Open Source Lab, Travis Whitehead
+===============================================================
```

Comment:
Please change this to using ``---`` level to match what I converted the other ones to.